src/quo/components/wallet/swap_input/style.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo.components.wallet.swap-input.style
+  (:require [quo.foundations.colors :as colors]
+            [quo.foundations.typography :as typography]))
+
+(defn- border-color
+  [theme]
+  (colors/theme-colors colors/neutral-10 colors/neutral-80 theme))
+
+(defn- loader-color
+  [theme]
+  (colors/theme-colors colors/neutral-5 colors/neutral-90 theme))
+
+(defn content
+  [theme]
+  {:border-width     1
+   :border-radius    16
+   :border-color     (border-color theme)
+   :background-color (colors/theme-colors colors/white colors/neutral-95 theme)})
+
+(defn row-1
+  [loading?]
+  {:padding        12
+   :gap            8
+   :align-items    (if loading? :center :flex-end)
+   :flex-direction :row})
+
+(defn row-1-loader
+  [theme]
+  {:width            74
+   :height           14
+   :border-radius    6
+   :background-color (loader-color theme)})
+
+(def input-container
+  {:flex           1
+   :flex-direction :row
+   :gap            5
+   :align-items    :flex-end})
+
+(defn input
+  [disabled? error? theme]
+  (merge {:font-size   27
+          :flex-shrink 1
+          :color       (if error?
+                         (colors/theme-colors colors/danger-50 colors/danger-60 theme)
+                         (if disabled?
+                           (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
+                           (colors/theme-colors colors/neutral-100
+                                                colors/white
+                                                theme)))
```

Comment:
I usually try to avoid nested `if`s as a regular practice, this one is really simple, but sometimes the `cond` looks better:

```clojure
(cond
  error?    (colors/theme-colors colors/danger-50 colors/danger-60 theme)
  disabled? (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
  :else     (colors/theme-colors colors/neutral-100 colors/white theme))
```

```

Reply:
for danger, warning etc prefer to use
`(colors/resolve-color :danger theme)` as this uses the color system then which allows it easier to extend modify later on 👍